tests/ops/qutrit/test_qutrit_parametric_ops.py

```diff
@@ -376,6 +431,9 @@ def test_differentiability(self, op, obs, grad_fn, phi, diff_method, tol):
 
         @qml.qnode(dev, diff_method=diff_method)
         def circuit(phi):
+            if issubclass(op, qml.TRZ):
```

Comment:
Do we care about subclasses? Couldn't we do
```suggestion
            if op is qml.TRZ:
```
?